kits/starter/c++/VariousSensors.cpp

```diff
@@ -47,6 +8,11 @@
 #include <iomanip>
 #include <string>
 #include <sstream>
+#include <ssd1327.h>
+#include <ssd1308.h>
+#include <ssd1306.h>
+#include <sainsmartks.h>
+#include <lcm1602.h>
```

Comment:
No need for a separate one, just force push the changes and this PR will reflect it. Thanks!